frontend/src/metabase/common/components/CollectionPicker/hooks.ts

```diff
@@ -51,10 +51,12 @@ export const useRootCollectionPickerItems = (
     currentUser?.personal_collection_id
       ? {
           id: currentUser?.personal_collection_id,
-          models: ["collection"],
+          models: ["collection", "dashboard"],
```

Comment:
I don't know that this is quite correct... This is making the assumption that we always care about dashboards in the collection picker, but we don't. (Eg: when creating a collection / dashboard / model).

However, I also don't know that it matters a ton, since as far as I can tell, we don't do a great job of deciding whether or not to show you these special collections either.

Reply:
Hmmmmm.... * sigh * you're right. I also didn't notice that was code reused elsewhere. It seems like this code would _only_ affect personal collections not other special collections, right?

I think I need to provide an option like `containerModels` that defaults to `["collection"]`. Does that sound good?

Reply:
I actually think my original assessment is correct. `models` should work... We don't need to only think about container models, but if there is anything clickable inside

Reply:
Wait, so are you considering this mergable or no?

Reply:
Talked to Nick, mergable, but ultimately not the final fix.

Reply:
Ultimately, it would be nice if the collection items endpoints supported `calculate_available_models`